client/src/boot/applyTransforms.js

```diff
@@ -65,6 +65,14 @@ const applyTransforms = () => {
       updater.form.addValidation(
         '*',
         (values, Validation, schema) => {
+          // Hardcoded exclusion for elemental inline forms
+          // This is done so that client-side validation doesn't run so that all validation is done
+          // on the server. This is done so that invalid, closed elements will popup open and
+          // also trigger a toast notification.
+          // Note that this has no effect on non-inline forms as they do not use form schema
+          if (schema.name.indexOf('ElementForm_') === 0) {
```

Comment:
This is to resolve https://github.com/silverstripe/silverstripe-elemental/pull/1214#issuecomment-2237727252

Reply:
This is kinda gross but it gets the job done. Would be better if we did something like `if (field.parents('.skipClientSideFormValidation').length > 0)` and add a `skipClientSideFormValidation` css class to the elemental form.

I'll let you decide if you want to do that - if not I'm okay with merging as is.

Reply:
I think merge as.

I've had a quick look at doing this "properly", would involved adding new API to formschema rather than just a class on a form.  Also because it deals with validation I think we'd possibly want to add it to the CompositeValidator, rather than directly on the Form e.g. Form->setClientSideValidation(false);  I'm not sure where it would be best.

Regardless it would be a proper standalone PR.  We could easily add this later should the need arise and remove the hack added here, so I think the hack here is OK

Reply:
Ahhh it's react nonsense. That explains why this doesn't happen for validation of fields directly on the page. I thought this was jquery nonsense but honestly it being react both makes sense and is hilarious.
No worries.